Makefile

```diff
@@ -41,8 +41,9 @@ build: install
 # Examples:
 #   make test
 #   make test GRUNT_FLAGS='--gruntfile=~/special/Gruntfile.js'
+#   make test GRUNT_UNIT_TEST_FLAGS='--browser=chrome' GRUNT_E2E_TEST_FLAG='--foo'
 test: build
 	hack/verify-dist.sh
-	hack/test-headless.sh test $(GRUNT_FLAGS)
-	hack/test-headless.sh test-integration $(GRUNT_FLAGS)
+	hack/test-headless.sh test-unit $(GRUNT_UNIT_TEST_FLAGS)
+	hack/test-headless.sh test-integration $(GRUNT_E2E_TEST_FLAGS)
```

Comment:
Cab we call them `GRUNT_INTEGRATION_TEST_FLAGS`? "e2e' is a different test suite.

Reply:
Yeah, I can rename that. I've actually been trying to rename "integration test" as "e2e test" over the past few PRs since thats really more accurate, but the label doesn't matter too much.